lib/reporters/cli/index.js

```diff
@@ -233,9 +235,24 @@ PostmanCLIReporter = function (emitter, reporterOptions, options) {
                 `${mime.mimeType}`,
                 `${mime.mimeFormat}`,
                 `${mime.charset}`
-            ].join(` ${colors.gray(symbols.star)} `);
-
+            ].join(` ${colors.gray(symbols.star)} `),
+            // eslint-disable-next-line max-len
+            bodyClipSize = BODY_CLIP_SIZE;
 
+        if (reporterOptions.truncateBodyOutput) {
+            // if infinite is passed to truncate-body-output
+            if (reporterOptions.truncateBodyOutput === 'infinite') {
```

Comment:
Why not take input as "infinity"? (Lowercase compare to ensure we don't need case sensitivity)

If you don't like the word infinity, then can use "none" too.

Reply:
I think infinity would be better since the description for --reporter-cli-truncate-body-output is specify the truncation size of body in verbose CLI output for requests and responses.